run-proxy

```diff
@@ -120,6 +126,34 @@ EOF
   fi
 fi
 
+if [ -n "$PROBE_LOCATION" ] || [ -n "$PROBE_LISTEN" ]; then
+  if [ -z "$PROBE_LOCATION" ] ; then
+    >&2 echo "--probe-location is required when --probe-listen is specified"
+    >&2 echo
+    >&2 echo "$HELP_STRING"
+    exit 1
+  fi
+
+  if [ -z "$PROBE_LISTEN" ] ; then
+    >&2 echo "--probe-listen is required when --probe-location is specified"
+    >&2 echo
+    >&2 echo "$HELP_STRING"
+    exit 1
+  fi
+
+  PROBE_CONFIG="$(cat <<EOF
+  server {
+    server_name ${HOSTNAME};
+    listen ${PROBE_LISTEN};
+
+    location ${PROBE_LOCATION} {
+      proxy_pass ${PROXY_PASS};
```

Comment:
That's is looking like an error.
1. We already have a distinct port for the probe, so we don't need to specify the location. We can do the probe on /.
2. On the other hand, we shall be able to select the destination for the probe, which is safe to be exposed.

Reply:
So I suggest:
1. remove `--probe-location` option
2. add `--probe-proxy-pass` option
3. use `--probe-proxy-pass` as a destination for proxy pass

Reply:
Done